contrib/pm-utils/40autorandr

```diff
@@ -5,6 +5,6 @@ exec > /var/log/autorandr.log 2>&1
 
 case "$1" in
 	thaw|resume)
-		autorandr --batch -c --default default
+		/usr/bin/autorandr --batch --change --default default
```

Comment:
The change is accompanied by a change in the Makefile,

    sed -i -re 's#/usr/bin/autorandr#$(subst #,\#,${PREFIX})/bin/autorandr#g' ${DESTDIR}/${PM_SLEEPHOOKS_DIR}/40autorandr

The general idea is that this allows users to install autorandr to a path outside of the system's $PATH.